Makefile

```diff
@@ -9,7 +9,7 @@ clean:
 
 lint:
 	poetry run black . --line-length=79
-	poetry run pycodestyle . --max-line-length=100
+	- poetry run pycodestyle . --max-line-length=100
```

Comment:
is there a way we can get pycodestyle to ignore a folder? that way we could get it to not run checks on the snapshot files that are auto generated?

Reply:
yeah but it fails on pyright ignore comments anyway